test/unit/deadline_client/cli/test_cli_manifest_diff.py

```diff
@@ -0,0 +1,106 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Integ tests for the CLI asset commands.
+"""
+import json
+import os
+from pathlib import Path
+from click.testing import CliRunner
+from deadline.client.cli._groups.manifest_group import cli_manifest
+import pytest
+import tempfile
+
+from deadline.client.cli import main
+
+
+@pytest.mark.skip("Random Failure with no credentials on Github")
+class TestSnapshot:
```

Comment:
There is something odd with calling CLI commands in unit test. As integration tests they pass but I am not sure why. I'm going to comment this out for now and then once submissions open I may move this over as an integ test.

Reply:
These look like integ tests to me. I'd imagine cli test tests the wrapper and mock call to the api (such as `_manifest_diff`).

Reply:
Yes, somehow there is something odd with them being unit tests. I will move them to be integ tests once the code is merged. I didn't want to merge in some functionality with potentially flaky integ tests.  (I'll follow up after this PR)